src/Provider/AzureMaps/AzureMaps.php

```diff
@@ -0,0 +1,312 @@
+<?php
```

Comment:
Actually, just realised you've got two copies of all the files here. Can you remove the ones in the root of the project please?